lib/tether/tether.go

```diff
@@ -252,6 +257,41 @@ func (t *tether) setMounts() error {
 			return fmt.Errorf("unsupported volume mount type for %s: %s", k, v.Source.Scheme)
 		}
 	}
+	return t.populateVolumes()
```

Comment:
Ah yes, I had tried this when I left this comment, but indeed from our conversation above if the `/` is left on the end you actually end up with the mount being bound to the target location. Sans the `/` works great! Sorry about that.